Correction regarding MBC2 RAM enable register (#586)
gbdev/pandocs: src/MBC2.md

#### When bit 8 is clear

When the least significant bit of the upper address byte is zero, the value that is written controls whether the RAM is enabled.

> When any value with `$A` in the lower 4 bits is written to this address range, RAM is enabled.

Reviewer: I think it sounds less confusing if the wording is flipped:

```suggestion
Save RAM will be enabled if and only if the lower 4 bits of the value written here are `$A`.
```
